aries-backchannels/afgo/afgo_backchannel.py

```diff
@@ -621,6 +625,80 @@ async def handle_agent_POST(self, command: BackchannelCommand):
 
         return (200, "{}")
 
+    async def handle_oob_v2_POST(self, command: BackchannelCommand):
```

Comment:
Not necessarily for this PR, but I created a way to build new topics in separate files to not make the backchannel file any larger than it already is. E.g. you can see an example for the mediation routes here: https://github.com/hyperledger/aries-agent-test-harness/blob/main/aries-backchannels/afgo/routes/mediation_routes.py

What I like about this approach is that it allows you to search for an endpoint and get the function where it is implemented (can be quite hard sometimes with the xxx_backchannel file). You can find everything related to a single topic in the file, and it gives you a separate handler per endpoint (instead of doing everything in once gigantic method).

I think it greatly reduces the complexity of the code, as we don't have to do all the topic translations etc... which hide what is actually happening.